examples/07_advanced/plot_haxby_mass_univariate.py

```diff
@@ -122,7 +122,8 @@
 
 # f_regression implicitly adds intercept
 _, pvals_bonferroni = f_regression(
-    grouped_fmri_masked, grouped_conditions_encoded
+    grouped_fmri_masked,
+    grouped_conditions_encoded.ravel(),
```

Comment:
unrelated but removes this sklearn warning

![image](https://github.com/user-attachments/assets/1901adb2-3f14-4db9-98ae-0d194b05d8e0)